querybook/webapp/components/CodeMirrorTooltip/SuggestionTooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button } from 'ui/Button/Button';
+
+interface IProps {
+    suggestionText: string;
+    onAcceptSuggestion: () => void;
+}
+
+const SUGGESTION_MAX_LENGTH = 20;
+
+export const SuggestionTooltip: React.FunctionComponent<IProps> = ({
+    suggestionText,
+    onAcceptSuggestion,
```

Comment:
have onAcceptSuggestion accept suggestionText so you can useCallback bind it in SuggestionTooltip